src/Resources/views/standard_layout.html.twig

```diff
@@ -101,6 +101,8 @@ file that was distributed with this source code.
                 {% if app.request.cookies.get('sonata_sidebar_hide') -%}
                     sidebar-collapse
                 {%- endif -%}"
+            {%- endblock -%}
+            {% block sonata_js_config %}                
```

Comment:
Not sure, if we need one more block... What about using this in your code:

```
{% block body_attributes -%}
{{ parent() }}
...
{% endblock %}
```

Reply:
Did you read my issue ?
Problem is not my code, but all others official bundles that need fix according to this change.
With this added block no need to update anything in the overrided bundles.

Reply:
I read the issue, but I don't like the solution for the fix. After this fix, we need some more PRs to modifiy the other confliction templates.

Can you try this solution: https://github.com/sonata-project/SonataAdminBundle/pull/5240#issuecomment-429342150

Reply:
:+1: Working with your solution

Reply:
I will modify 
Should I encapsulate in a block if someone would override this part ?
I can reuse the sonata_javascript_config block.
What do you think ?

Reply:
I am against a new block, if not needed 👎

Reply:
PR updated